cirrus/server/tests/test_telemetry.py

```diff
@@ -105,3 +108,55 @@ async def test_enrollment_metrics_recorded_with_compute_features(mocker, recipes
 
     assert ping_spy.call_count == 1
     assert app.state.metrics.cirrus_events.enrollment.test_get_value() is None
+
+
+@pytest.mark.asyncio
+async def test_enrollment_status_metrics_recorded_with_metrics_handler(mocker, recipes):
+    _, app.state.metrics = initialize_glean()
```

Comment:
I'm personally not a fan of destructuring with _ in python because it actually declares a variable named _ (unlike discarding immediately like in rust), but I'm not gonna block this pr on that.

Reply:
Yeah it's a pattern rather than an actual syntax in Python unfortunately.  The [docs](https://docs.python.org/3/reference/lexical_analysis.html#reserved-classes-of-identifiers) do refer to it but it's not explicitly supported as an actual syntax for unused variables outside of the wildcard in match statements.  But using `_` to denote an unused variable is a widely accepted pattern in Python so I think it's reasonable to use here.

Funnily, I found a rejected [PEP640](https://peps.python.org/pep-0640/) suggesting using `?` as an explicit unused variable identifier.  I can see why it was rejected.